src/project-manager.ts

```diff
@@ -385,7 +385,11 @@ export class ProjectConfiguration {
 
             const info: PluginCreateInfo = {
                 config: configEntry,
-                project: { projectService: { logger: this.logger } }, // TODO: may need more support
+                project: {
+                    // TODO: may need more support
+                    getCurrentDirectory: this.getHost().getCurrentDirectory.bind(this),
```

Comment:
bind() is not type safe, prefer wrapping in a function manually

Reply:
:heavy_check_mark: